### ISSUE TITLE
Support for importing a TypeScript module's default export

### ISSUE THREAD

**Issue by schmod** [dotansimha/graphql-code-generator#1997]
**Is your feature request related to a problem? Please describe.**

I would like to import types/classes (to be used as custom mapped types, contexts, and root-types) that are exported from a TypeScript module via `export default`.

**Describe the solution you'd like**

I would like to be able to import mapped types, contexts, root-types, etc with a syntax that looks something like this:

```yaml
plugins
  config:
    contextType: ./my-context#default
```

where `my-context.ts` looks like:
```ts
export default class MyContext{}
```

**Describe alternatives you've considered**
N/A -- I'm open to alternatives to the above syntax.

**Additional context**
This pattern is commonly seen in ORMs (such as [sequelize-typescript](https://github.com/RobinBuschmann/sequelize-typescript#readme)) that expect model-definitions to be exposed via default exports.

**Comment by dotansimha** [comment]
Hi @schmod , that's sounds like a good idea.
I added it here: https://github.com/dotansimha/graphql-code-generator/pull/2014

**Comment by dotansimha** [comment]
Available in 1.3.0 🎉
